staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -116,8 +119,51 @@ service RuntimeService {
     rpc Status(StatusRequest) returns (StatusResponse) {}
 }
 
+// GetEventsFilter is used to filter a list of events.
+// All those fields are combined with 'AND'
+message GetEventsFilter {
+    // ID of the container, sandbox or image.
+    string id = 1;
+    // LabelSelector to select matches.
+    // Only api.MatchLabels is supported for now and the requirements
+    // are ANDed. MatchExpressions is not supported yet.
+    map<string, string> label_selector = 3;
+}
+
+message GetEventsRequest {
+    // GetEventsFilter to filter a list of events.
+    GetEventsFilter filter = 1;
+}
+
+message ContainerEventResponse {
+    // ID of the container
+	string container_id = 1;
+
+	// Type of the container event
+    ContainerEventType container_event_type = 2;
+
+    // Creation timestamp of this event
+    int64 created_at = 3;
+
+    // ID of the sandbox container
+	string sandbox_id = 4;
+}
+
+enum ContainerEventType {
+    // Container created
+    CREATED = 0;
+
+    // Container started
+    STARTED = 1;
+
+    // Container Stopped
+    STOPPED = 2;
```

Comment:
Do we need  DELETED here as well? 

Even though kubelet usually initiates the delete but as the draft KEP (https://hackmd.io/Spy1RLUzRqGbYtmOOZEKBw#Proposal) mentions, it is also possible that the user interacts with runtime directly through crictl etc.

Reply:
Sounds like a good idea, but do we intent to support actions taken on the containers outside of kubelet? (e.g. by `crictl`)

Reply:
We need to support this case IMO because currently the relist-based PLEG handles this case correctly, and we don't want to cause any regression by moving to event-baed PLEG.

Reply:
@ruiwen-zhao I updated the PR that add `DELETED`. I wasn't simply able to add `DELETED` because was running into, 

```
api.proto:202:9: "DELETED" is already defined in "runtime.v1".
api.proto:202:9: Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  Therefore, "DELETED" must be unique within "runtime.v1", not just within "ImageEventType".
```

So I added `CONTAINER_DELETED_EVENT` and updated the rest of the events accordingly.